pkg/config/configobserver/configobserver.go

```diff
@@ -130,14 +130,15 @@ func (c *Controller) retrieveConfig() error {
 		nextConfig.Report = len(nextConfig.Endpoint) > 0
 
 		if intervalString, ok := secret.Data["interval"]; ok {
-			duration, err := time.ParseDuration(string(intervalString))
-			if err == nil && duration < time.Minute {
-				err = fmt.Errorf("too short")
+			duration, errD := time.ParseDuration(string(intervalString))
```

Comment:
The `err` variable was not propagated, so we lost it outside of this block

Reply:
style: we use `errp` in Go for these sorts of things (single lowercase letter after `err`)

Reply:
You would lose it outside the block anyway - errD is local.

Reply:
Yes, so was `err` in this case and that's why we were loosing the message: the point of this change, to make it local explicitly, and set it to the global `err` here https://github.com/openshift/insights-operator/pull/21/files#diff-e217a6a94e2f38d09ce7df512a06b266R141.

Btw. we would be able to avoid the `errp` workaround if we used the "return early" style, which makes me think why it's not that way in the first place and the only reason

Reply:
Thinking about it a bit more, it looks using "return early" approach would avoid us having to care about the scope of `err` and simplifies the method a bit.